### PR TITLE
Documentation updates: add key shortcuts, fix a few typos, and remove dead link

### PR DESCRIPTION
**Pull request by cscorley**

### CODE REVIEW

File: doc/Default_hotkeys.md [hunk 1]
```diff
@@ -51,99 +61,127 @@ window list by using the first parameter.
...
+
 _Move and resize_ the active window _to_ the size of the _work area_.
 
 > This implicitly sets the window to be floating.
 
 -------------------------------------------------------------------------------
 `Config_hotkey=!<n>::View_moveWindow(<n>)`
 
-Manually _move_ the active window _to the n<sup><small>th</small></sup> area_ 
+<kbd>Alt</kbd>&lt;<kbd>n</kbd>&gt;
```
Comment (cscorley) [inline doc/Default_hotkeys.md]: Was quite unsure how to best label this "key", so I took what I thought would be a good compromise: surrounding the n key with the < and >. Can move them inside all the same.

File: doc/Home.md [hunk 2]
```diff
@@ -64,10 +64,6 @@ description.
 configuration. With this overview you may find unused keys, which can be used
 for non-bug.n hotkeys or to reassign bug.n functions.
 
-##### [Xmonad hotkeys](./Xmonad-hotkeys.md)
-> A list of hotkeys, which are similar to those used in
-[xmonad](http://xmonad.org/), a dynamically tiling X11 window manager on *n?x.
-
```
Comment (cscorley) [inline doc/Home.md]: It seems that this file no longer exists, and is contained in the following file

Comment (alphaCTzo7G) [reply doc/Home.md]: agree.. I noticed this too. The xmonad config file is not in here.. but there are config files in the user suggested hotkeys section..

File: doc/Default_hotkeys.md (hunk 1 above)
Comment (joten) [reply doc/Default_hotkeys.md]: The [Cheat sheet](https://github.com/fuhsjr00/bug.n/tree/master/doc/Cheat_sheet) is dependent on doc/Default_hotkeys.md. If the creation process of the cheat sheet is affected, either this or doc/Default_hotkeys.md should be adjusted.

**Comment by joten** [comment]
I adopted the fixes (typos, spaces in between abbreviations and the dead link), but I thought the additional <kbd>-lines reduce the readability of the md-file and would be redundant to the cheat sheet.

But thanks for the review and correction of the documentation.

**Pull request closed without merging**